cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -1017,13 +994,13 @@ features:
           DUPLICATE: Failed to duplicate connection
           EDIT: Failed to edit connection
         ModalHeader:
-          ADD: "Add Connection: Big Query"
-          DUPLICATE: "Duplicate Connection: {connection}"
-          EDIT: "Edit Connection: {connection}"
+          ADD: "Add connection: Big Query"
+          DUPLICATE: "Duplicate connection: {connection}"
+          EDIT: "Edit connection: {connection}"
         name: Name
         projectId: Project ID
         required: Required
-        serviceAccountKeyfile: Service Account Keyfile Location
+        serviceAccountKeyfile: Service account keyfile location
```

Comment:
where is this used? keyfile is not a word. `Service account key file location`?

Reply:
These are used when creating BigQuery and GCS connections I think